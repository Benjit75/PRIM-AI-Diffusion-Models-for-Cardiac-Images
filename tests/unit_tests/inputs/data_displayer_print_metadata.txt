{'data_name': 'test', 'id_example': 'patient101', 'indentation': '----'}
{'data_name': 'train', 'id_example': 'patient002', 'indentation': '####'}