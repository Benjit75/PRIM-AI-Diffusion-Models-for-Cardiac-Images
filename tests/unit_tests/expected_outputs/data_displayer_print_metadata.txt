----ID: patient101, height: 169.0cm, weight: 79.0kg, group: DCM, nb_frames: 30
####ID: patient002, height: 160.0cm, weight: 70.0kg, group: DCM, nb_frames: 30